packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselAutoplayControlled.stories.tsx

```diff
@@ -78,9 +78,9 @@ export const AutoplayControlled = () => {
         >
           <CarouselButton navType="prev" />
           <CarouselAutoplayButton
-            autoplay={enableAutoplay}
-            onClick={() => {
-              setEnableAutoplay(!enableAutoplay);
+            checked={enableAutoplay}
+            onCheckedChange={(_, data) => {
+              setEnableAutoplay(data.autoplay);
```

Comment:
```suggestion
              setEnableAutoplay(data.checked);
```

Let's follow the naming of `props`